logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/SessionEstablisher.kt

```diff
@@ -14,7 +14,7 @@ import com.wire.kalium.logic.data.user.UserId
 import com.wire.kalium.logic.functional.Either
 import com.wire.kalium.logic.functional.suspending
 import com.wire.kalium.logic.wrapCryptoRequest
-import com.wire.kalium.cryptography.UserId as CryptoUserId
+import com.wire.kalium.cryptography.PlainUserId as CryptoUserId
```

Comment:
I think we can remove the alias now that it was renamed to `PlainUserId`

Reply:
Done